src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/HistoricalBeneficialOwnerValidator.java

```diff
@@ -27,8 +27,11 @@ public Errors validate(List<TrustDataDto> trustDataDtoList, Errors errors, Strin
             List<HistoricalBeneficialOwnerDto> historicalBeneficialOwners = trustDataDto
                     .getHistoricalBeneficialOwners();
             if (!CollectionUtils.isEmpty(historicalBeneficialOwners)) {
+                boolean isUnableToObtainAllTrustInfo = trustDataDto.getUnableToObtainAllTrustInfo();
                 for (HistoricalBeneficialOwnerDto historicalBeneficialOwnerDto : historicalBeneficialOwners) {
-                    validateCeasedDate(historicalBeneficialOwnerDto.getCeasedDate(), errors, loggingContext);
+                    if (!isUnableToObtainAllTrustInfo) {
+                        validateCeasedDate(historicalBeneficialOwnerDto.getCeasedDate(), errors, loggingContext);
```

Comment:
if unableToObtainAllTrustInfo is true then i think it effectively makes makes the cease date optional, so i think we will still need to validate it if unableToObtainAllTrustInfo is true and the ceasedate is not empty

Reply:
Updated to validate ceased date if it is not null